### PR TITLE
Chef server fqdn and LB added in config.toml

### PR DESCRIPTION
**Pull request by SanjuPal01**
### :nut_and_bolt: Description: What code changed, and why?
As a automate HA developer, i want to accept chef-server fqdn and lb_root-ca as input in config.toml. These changes should be available for all kinds of deployment topologies.
Also make automate root_ca independent from custom_certs_enabled flag.

### :chains: Related Resources
https://chefio.atlassian.net/browse/CHEF-3631

### :athletic_shoe: How to Build and Test the Change
Checkout to this branch and build `automate-cli` and `automate-backend-deployment`(automate-ha-deployment).
OR
you can use these packages: 
`ssanju/automate-cli/0.1.0/20230630064113`
`ssanju/automate-ha-deployment/0.1.0/20230628074623`

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
Screenshot 2023-06-30 at 6 42 30 PM
Screenshot 2023-06-30 at 6 50 54 PM

### CODE REVIEW

File: components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go [hunk 1]
```diff
@@ -77,6 +77,12 @@ enable_custom_certs = false
 [chef_server.config]
 ## === INPUT NEEDED ===
 
+# Chefserver Load Balancer FQDN eg.: "chefserver.example.com"
```
Comment (vivek-yadav) [inline components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go:80]: I think this also needs to be added to Docs also in Sample Config and other places

File: components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go [hunk 2]
```diff
@@ -52,6 +52,9 @@ admin_password = ""
 # Automate Load Balancer FQDN eg.: "chefautomate.example.com"
 fqdn = ""
 
+# Add Automate Load Balancer root-ca
```
Comment (vivek-yadav) [inline components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go:55]: I think this also needs to be added to Docs also in Sample Config and other places

File: components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go (hunk 1 above)
Comment (SanjuPal01) [reply components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go:80]: will raise separate PR for this

File: components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go (hunk 2 above)
Comment (SanjuPal01) [reply components/automate-cli/cmd/chef-automate/initConfigHaTmpl.go:55]: will raise separate PR for this

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**